Source/QuantityTypes.Tests/Quantities/AngleTests.cs

```diff
@@ -54,5 +54,15 @@ public void ToString_NorwegianCulture_CheckSpaces()
             var a = 90 * Angle.Degree;
             Assert.AreEqual("90,0°", a.ToString("0.0 [°]", CultureInfos.Norwegian));
         }
+
+        [Test]
+        public void ParseStringToAngleType()

```

Comment:
Yes, we need use reflection, which is exactly the same as we do in PropertyTools. We want to get the parse method by reflection.